features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/test/Comments/Comments.test.js

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { unwrap } from '@material-ui/core/test-utils';
+import { Typography } from '@material-ui/core';
+import ArrowDropDown from '@material-ui/icons/ArrowDropDown';
+import ArrowDropDownCircleOutlined from '@material-ui/icons/ArrowDropDownCircleOutlined';
+import Comments from '../../src/app/components/Apis/Details/Comments/Comments';
+import Comment from '../../src/app/components/Apis/Details/Comments/Comment';
+import CommentAdd from '../../src/app/components/Apis/Details/Comments/CommentAdd';
+
+const CommentsUnwrapped = unwrap(Comments);
+
+let wrapper;
+const theme = {
+    custom: {
+        commentsLimit: 5
+    }
+};
+const api = {
+    id: '6e770272-212b-404e-ab9c-333fdba02f2f',
+    isDefaultVersion: true,
+    lastUpdatedTime: '2018-11-27T03:08:22.569Z',
+    lifeCycleStatus: 'Published',
+    name: 'Swagger Petstore'
+};
+
+beforeEach(() => {
+    wrapper = shallow(<CommentsUnwrapped api={api} classes={{}} theme={theme} /> );
+});
+
+beforeAll(() => {
+    CommentsUnwrapped.prototype.componentDidMount = () => {
+        console.log('componentDidMount method is called');
```

Comment:
can we remove this console statement ?

Reply:
Actually this will be printed when we run tests. It will not print anything in the brower console. It will print 'componentDidMount method is called' in the command line which we run tests in order to know that the componentDidMount method is called in a mocked way.

Reply:
noted, can resolve this